GhostFSLauncher.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+export LD_LIBRARY_PATH=/usr/local/lib/libssl.1.1
+./GhostFS "$@"
```

Comment:
you could also `exec ./GhostFS "$@"` to get rid of that useless bash process.

Reply:
https://github.com/HIP-infrastructure/ghostfs/pull/8